config/ftbquests/normal/chapters/08d906a0/2c4c4fee.snbt

```diff
@@ -11,5 +11,15 @@
 		items: [{
 			item: "minecraft:emerald"
 		}]
+	}],
+	rewards: [{
+		uid: "fe7357ff",
+		type: "item",
+		item: {
+			id: "ftbquests:lootcrate",
```

Comment:
Please remove all FTB loot crates from Skyblock quests rewards. I believe that player should never receive Loot Crates from optional quests.